application/frontend/web/index.php

```diff
@@ -12,7 +12,7 @@
      * Load environment config if present, else expect local.php
      */
     $appEnv = \Sil\PhpEnv\Env::get('APP_ENV', 'production');
-    if (! in_array($appEnv, ['development', 'dev', 'staging', 'stage', 'stg', 'production', 'prod', 'prd'])) {
+    if (! in_array($appEnv, ['test', 'development', 'dev', 'staging', 'stage', 'stg', 'production', 'prod', 'prd'])) {
```

Comment:
Not directly related to this change, but it seems to generally be a good practice when using [in_array(...)](http://php.net/in_array) to use this form...
```php
in_array($needle, $haystack, true)
```
... supplying that last `true` to force a strict comparison, rather than this form...
```php
in_array($needle, $haystack)
```
... because the latter can "match" on a boolean true (because `true == 'any non empty string'`, but `true !== 'any string'`).

Reply:
Good suggestion, thanks.